frontend/ui/data/creoptions.lua

```diff
@@ -591,11 +591,7 @@ Note that your selected font size is not affected by this setting.]]),
                                     0.95, 0.98, 1, 1.02, 1.05, 1.1, 1.15, 1.2, 1.25, 1.3, 1.35, 1.4, 1.45,
                                     1.5, 1.6, 1.7, 1.8, 1.9, 2, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9,
                                     3, 3.5, 4, 4.5, 5, 5.5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 },
-                    args_table = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12,
-                                   13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25,
-                                   26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 29, 40,
-                                   41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56 },
-                    value_step = 1,
+                    value_table_shift = 1, -- crengine counts index from 0
```

Comment:
I know this works. But reading this is a bit confusing: if crengine counts index from 0, I would expect value_table_shift=-1. You use `+ table_shift` or `- table_shift` in various bits of code. Dunno which of -1 or +1 would make more sense in the code where it is used.
So, you might add a more explicite comment. Improvising, dunno if my quick reading/understanding is right:
_The setting is saved starting from index 0, and provided as-is to crengine (which expects index from 0). So, we will add/substract this shift anywhere we just access value_table to get a value to display._